pkg/event/target/mqtt.go

```diff
@@ -96,7 +101,7 @@ func (target *MQTTTarget) Send(eventData event.Event) error {
 
 	token := target.client.Publish(target.args.Topic, target.args.QoS, false, string(data))
 
-	if token.Wait() {
+	if token.WaitTimeout(150 * time.Millisecond) {
```

Comment:
Could you add a comment explaining why you are returning error only when token hadn't timed out and ignore the error otherwise?

Reply:
yes. done

Reply:
How did you find 150ms is good enough for event delivery?  Is it specified in MQTT?

Reply:
No it is just an assumption

Reply:
What happens if the message is not delivered in 150ms?  Does it return timeout error?

If timeout is important to have, I think it should be configurable.  Also if MQTT store directory is used, is it required to wait for message delivery?